packages/signers/src/turnkey/types.ts

```diff
@@ -0,0 +1,20 @@
+import type { TStamper } from "@turnkey/http/dist/base";
+import type { Transport } from "viem";
+
+export interface TurnkeyAuthParams {
```

Comment:
out of curiosity, are these types and/or fields in some base/core JS SDK that Turnkey has that can be imported? would love to not have to maintain these if there's one source of truth :)

Reply:
currently no. but i totally agree and we may be making some changes here in the near future by importing a type from the turnkey sdk. more in this comment: https://github.com/alchemyplatform/aa-sdk/pull/292#discussion_r1415978792